package.json

```diff
@@ -115,7 +116,7 @@
     "prop-types": "^15.7.2",
     "react": "18.3.1",
     "react-dom": "18.3.1",
-    "webex": "^2.59.3"
+    "webex": "^2.60.2"
```

Comment:
Why do we have `webex` in the dependency as well as devDependency?

Reply:
@rarajes2  This `webex` is there in the peer-dependency not in the devDependency